2.0/x-oca-event.md

```diff
@@ -2,47 +2,55 @@
+| cross_process_target_ref | `object-ref`                | references the target process in case of a cross process event. must be of type `process`           |
+| domain_ref               | `object-ref`                | references a domain related to this event. must be of type `domain-name`                            |
+| registry_ref             | `object-ref`                | references windows registry data related to this event. must be of type `windows-registry-key`      |
+| network_ref              | `object-ref`                | references network traffic related to this event. must be of type `network-traffic`                 |
+| ip_refs                  | `list` of type `object-ref` | references any ip addresses related to this event. must be of type `ipv4-address` or `ipv6-address` |
+| user_ref                 | `object-ref`                | references the user related to this event. must be of type `user-account`                           |
+| severity                 | `number`                    | Severity of the event on a scale of 1-100.                                                          | 
+| start                    | `timestamp`                 | The start time in UTC of the event.                                                                 |
+| end                      | `timestamp`                 | The end time in UTC of the event.                                                                   |
+| timezone                 | `string`                    | The timezone where the event was discovered.                                                        |
+| duration                 | `string`                    | The duration of the event.                                                                          |
+| dataset                  | `string`                    | The dataset of the event.                                                                           |
+
+### Event Specific properties
+Certain events are important for security context which have their unique properties. The `x-oca-event` supports the following optional properties that should be populated only in certain `action` types.
+
+| property name | type     | description                                                                                                                                                                                                                           |
+|---------------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| pipe_name     | `string` | In windows, there are many security rules that look for the pipe name property in a named pipe event. the value of this property should include the full pipe name in case of a named pipe event. see [example](#a-named-pipe-event). |
```

Comment:
Couldn't this just go in the main table? I think most of the properties on the x-oca-event are optional.

Reply:
it could, I wanted to emphasie it was relevant only for certain event types and maybe others will follow. Maybe its an overkill

Reply:
I think the description is fine. Do you have an example of a security product that would return data that could be mapped to this property?

Reply:
I merged it within the main table